.travis.yml

```diff
@@ -43,3 +43,20 @@ script:
 after_success:
 - bash <(curl -s https://codecov.io/bash)
 - make docker
+before_deploy:
+- make dist
+deploy:
+  provider: releases
+  api_key:
+    secure: TODO
```

Comment:
Do I need to do anything to allow for a token?

Reply:
Oh, I need to remember how to encrypt the github token. Do you have one handy still?